packages/ses/src/lockdown-shim.js

```diff
@@ -129,23 +144,45 @@ export const repairIntrinsics = (
   // [`stackFiltering` options](https://github.com/Agoric/SES-shim/blob/master/packages/ses/lockdown-options.md#stackfiltering-options)
   // for an explanation.
   options = /** @type {LockdownOptions} */ ({ ...firstOptions, ...options });
+
+  const {
+    getEnvironmentOption: getenv,
+    capturedEnvironmentOptionNames,
+  } = makeEnvironmentCaptor();
+
   const {
+    errorTaming = getenv('LOCKDOWN_ERROR_TAMING', 'safe'),
+    errorTrapping = getenv('LOCKDOWN_ERROR_TRAPPING', 'platform'),
+    regExpTaming = getenv('LOCKDOWN_REGEXP_TAMING', 'safe'),
+    localeTaming = getenv('LOCKDOWN_LOCALE_TAMING', 'safe'),
+    consoleTaming = getenv('LOCKDOWN_CONSOLE_TAMING', 'safe'),
+    overrideTaming = getenv('LOCKDOWN_OVERRIDE_TAMING', 'moderate'),
+    stackFiltering = getenv('LOCKDOWN_STACK_FILTERING', 'concise'),
+    // TODO domainTaming should change to safe-by-default in the next breaking relase.
+    domainTaming = getenv('LOCKDOWN_DOMAIN_TAMING', 'unsafe'),
+    overrideDebug = arrayFilter(
+      stringSplit(getenv('LOCKDOWN_OVERRIDE_DEBUG', ''), ','),
+      stringIsNotEmpty,
+    ),
```

Comment:
This is slightly weak since it doesn’t distinguish a defined by empty environment variable, which should be invalid but probably doesn't matter.